app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3752,8 +3752,9 @@ public boolean onKeyDown(final int keyCode) {
             default:
                 break;
             case KeyEvent.KEYCODE_SPACE:
-                if (isFullscreen) {
```

Comment:
Not sure why this check existed here in the first place, what can/should happen when pressing Space outside of full screen? (typing in the search or comment box?)
If in doubt, could just leave the original if() check, and just add the if (isPlaying) part.

Reply:
See #5331. In that PR there doesn't seem to be any explanation about why the `if (isFullscreen)`. So I think we can safely remove it, what do you think @mbarashkov?

Reply:
@Stypox In that PR, I had to do this because not in full screen space key was used on TV's to navigate the screen vertically, as I recall.

Reply:
Ok, that makes sense, thank you. Then I'll restore the previous `if (isFullscreen)`.